src/User32.Desktop/User32+ScanCode.cs

```diff
@@ -0,0 +1,193 @@
+﻿// Copyright (c) to owners found in https://github.com/AArnott/pinvoke/blob/master/COPYRIGHT.md. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+
+namespace PInvoke
+{
+    using System;
+
+    /// <content>
+    /// Contains the Keyboard scan code constants.
+    /// </content>
+    public partial class User32
+    {
+        /// <summary>
+        /// General keyboard scan code constants on the same order that it can be found on <see cref="VirtualKey" /> constants
+        /// </summary>
+        /// <remarks>Scan codes are device-dependant values, thesse are general values used by most keyboards.</remarks>
+        public enum ScanCode : short
+        {
+            LBUTTON = 0,
+            RBUTTON = 0,
```

Comment:
Just fixed that. The name of the keys were meant to reflect the VK_* constant names, but for scan codes. Unfortunately, many VK codes have no corresponding generic scan code (they are a bundle of one or more non-default scan codes), thus were set to zero.

I added explanation above and reordered + trimmed all the zeroes but one (NONAME).